bazel/packaging/BUILD

```diff
@@ -75,9 +74,7 @@ native_package(
     cc_binaries = [
         "//src/v/kafka/client/direct_consumer/verifier:direct_consumer_verifier",
     ],
-    include_sysroot_libs = True,
-    install_path = ":install_path",
-    rpath_override = "$ORIGIN/../lib",
+    install_path = "/opt/redpanda_installs/direct_consumer_verifier",
```

Comment:
Is this package used anywhere? If so should it install somewhere else?

Reply:
@rockwotj sorry I only saw after I had merged.

This package is used in DT CI runs. It should install the same place as `direct_consumer_verifier_ducktape` which is only used for local dev DT (this is the "split" we were talking about in slack).

Reply:
The naming in this file is unclear: the _ducktape packages are _only_ used for local dev use, the dev-only `run-bazel-ducktape-tests` target. The _tar targets are the ones used in ducktape by CI.